### PR TITLE
Clean up top level Ray dir

### PR DESCRIPTION
**Pull request by ericl**
## Linter

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

File: setup_hooks.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-ln -s $PWD/hooks/pre-push $PWD/.git/hooks/pre-push
+ln -s $PWD/doc/dev/hooks/pre-push $PWD/.git/hooks/pre-push
```
Comment (edoakes) [inline setup_hooks.sh]: Not sure how I feel about this being moved to `doc`, maybe `tools` or `scripts` is better?

**Review by edoakes — APPROVED** [review]
LGTM, big improvement. Would prefer not to have the hook script in `doc` but ok either way.

File: setup_hooks.sh (hunk 1 above)
Comment (richardliaw) [reply setup_hooks.sh]: @ericl Agree with @edoakes, can you move it to `scripts/`?

File: doc/source/tune-searchalg.rst [hunk 2]
```diff
@@ -43,7 +43,7 @@ In order to use this search algorithm, you will need to install Bayesian Optimiz
 
     $ pip install bayesian-optimization
 
-This algorithm requires `setting a search space and defining a utility function <https://github.com/fmfn/BayesianOptimization/blob/master/examples/advanced-tour.ipynb>`__. You can use BayesOptSearch like follows:
+This algorithm requires `setting a search space and defining a utility function <https://github.com/fmfn/BayesianOptimization/blob/master/doc/examples/advanced-tour.ipynb>`__. You can use BayesOptSearch like follows:
```
Comment (richardliaw) [inline doc/source/tune-searchalg.rst]: this is not supposed to be here... it's an external library

**Pull request merged**